CMakeLists.txt

```diff
@@ -272,7 +272,7 @@ function(glob_generate target file_glob)
 		if (${file_ext} STREQUAL "jinja")
 			add_custom_command(OUTPUT ${out_file}
 				COMMAND
-					${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/scripts/jinja_gen.py ${in_file} ${CMAKE_CURRENT_SOURCE_DIR}
+					${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/scripts/jinja_gen.py ${in_file} ${CMAKE_CURRENT_SOURCE_DIR} --ros_distro $ENV{ROS_DISTRO}
```

Comment:
Not sure if I am missing something, but this seems problematic. There would be a corner case when you have a ROS distro installed on your system, but you are trying to run it without ROS e.g. `make px4_sitl gazebo` it will still generate the models with the ROS plugins right?